codechefcli/problems.py

```diff
@@ -233,6 +236,72 @@ def search_problems(search_type):
         print(SERVER_DOWN_MSG)
 
 
+def get_tags(tags):
+    """
+    :desc: print all the tags and all the problems related to the combination of tags specified.
+    :param: list of the tags, whose combination is to be searched for problems.
+    """
+    if len(tags) == 0:
+        print_tags()
+    else:
+        print_problem_tags(tags)
+
+
+def print_tags():
+    """
+    :desc: prints all the tags
+    """
+    session = get_session()
+    url = BASE_URL + '/get/tags/problems'
+    req_obj = request(session, 'GET', url)
+    if req_obj.status_code == 200:
+        all_tags = json.loads(req_obj.text)
+        data_rows = []
+        temp = []
+        for en, all_tag in enumerate(all_tags):
+            if ((en + 1) % 6 != 0):
+                temp.append(all_tag['tag'])
+            else:
+                data_rows.append(temp)
+                temp = []
+        print_table(data_rows)
+
+    elif req_obj.status_code == 503:
+        print(SERVER_DOWN_MSG)
+
+
+def print_problem_tags(tags):
+    """
+    :desc: prints all the problems containg the list of tags.
+    :params: list of all the tags
+    """
+    session = get_session()
+    url = BASE_URL + '/get/tags/problems/' + ','.join(tags)
+    req_obj = request(session, 'GET', url)
+    if req_obj.status_code == 200:
+        all_tags = json.loads(req_obj.text)
+        data_rows = [[HEADINGS['code'], HEADINGS['name'],
```

Comment:
HEADINGS should be a list such as `['CODE', ...]`. And, choose proper name for `HEADINGS`.